client/src/pages/Profile/useStyles.ts

```diff
@@ -0,0 +1,64 @@
+import { makeStyles } from '@material-ui/core/styles';
+const useStyles = makeStyles((theme) => ({
+  profileContainer: {
+    margin: theme.spacing(4),
+    height: '900px',
+    maxWidth: '760px',
+  },
+  coverImage: {
+    maxHeight: '260px',
+    width: '100%',
+    borderRadius: theme.shape.borderRadius,
+  },
+  basicInfoContainer: {
+    position: 'relative',
+    bottom: '94px',
+  },
+  profilePic: {
+    borderRadius: '50%',
+    maxWidth: '180px',
+    height: '180px',
+    width: 'auto',
+    border: '2px solid white',
+    boxShadow: '-1px 9px 11px -5px rgb(0 0 0 / 35%)',
+  },
+  locationIcon: {
+    position: 'relative',
+    top: '5px',
+  },
+  aboutContainer: {
+    margin: theme.spacing(4),
+    position: 'relative',
+    bottom: '90px',
+  },
+  description: {
+    maxWidth: '680px',
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+  galleryContainer: {
+    flexDirection: 'row',
+  },
+  galleryPic: {
+    margin: theme.spacing(1),
+    height: '180px',
```

Comment:
Optional: When we assign a fixed height/width to images, they lose their aspect ratio. Personally I like to add `objectFit: "cover"` so that they aspect ratio is preserved and the images look better :)

Reply:
Thanks  for the Tip :)